docs/src/installation/deployment.md

```diff
@@ -574,11 +574,68 @@ If you decided to set up a dex server, you can parametrize the [metal role](http
+- deployable via CI/CD jobs
+- accessible from the internet from specific IPs
+
+In order to accomplish this task remotely and in a nearly automatic manner, you have to bootstrap the components in this order:
+
+1. Management Firewalls
+2. Management Servers
+3. Management Spines
+4. Management Leaves
+5. Leaves, Spines and Exits
+
+This document assumes that all cabling is done. Here is a quick overview of the architecture:
+
+![Management Net](mgmt_net_layer3.svg)
+
+### Management Firewalls
+
+As you can see, the Management Firewalls are the first bastion hosts in a partition to provide access to our infrastructure. There are two of them in each partition to guarantee high availability and load balancing. The very first configuration of these routers has to be done manually to solve the chicken and egg problem that you need the management firewalls in place to deploy the partition. Manually means that we generate a configuration template with ansible that we deploy with copy/paste, and the load, through the machine console. Once the management server has been deployed, we are able to deploy this configuration via gitlab-runner and ansible. For this you need the user and the ssh-key, which is deployed with the configuration file mentioned above.
+The Edgerouters has to fulfill some requirements including:
+
+- provide and restrict access to the management network from the internet with a firewall ruleset
+- provide destination NAT to the management server and its IPMI interface (public IP access from a specific IP address only)
+- provide onieboot and ztp via dhcp options for the management spine
+- provide dhcp management addresses for management spine, management server and ipmi interface of the management server
+- Hairpin-NAT for the management server to access itself via its puplic IP, needed by the gitlab-runner to delegate CI-Jobs.
+- propagate a default gateway via BGP
 
-Documentation will follow soon. Please come to our Slack channel when you have come to this point and require any help.
+### Management Servers
+
+The second bastion hosts are the management servers. They are the main bootstrapping components of the management network. They also act as jump hosts for all components in a partition. Once they are installed and deployed, we are able to bootstrap all the other components. To bootstrap the management servers, we generate an ISO image which will automatically install an OS and an ansible user with ssh keys. It is preconfigured with a preseed file to allow an unattended OS installation for our needs. This is why we need remote access to the IPMI interface of the Management Servers: The generated ISO is attached via the virtual media function of the BMC. Ater that, all we have to do is boot from that virtual CD-ROM and wait for the installation to finish. Deployment jobs (Gitlab-CI) in a partition are delegated to the appropriate Management Servers, therefore we need a gitlabci-runner active on each management server.
```

Comment:
```suggestion
The second bastion hosts are the management servers. They are the main bootstrapping components of the management network. They also act as jump hosts for all components in a partition. Once they are installed and deployed, we are able to bootstrap all the other components. 

To bootstrap the management servers, we generate an ISO image which will automatically install an OS and an Ansible user including SSH keys. It is preconfigured with a preseed file to allow an unattended OS installation for our needs. This is why we need remote access to the IPMI interface of the management servers: The generated ISO is attached via the virtual media function of the BMC. After that, all we have to do is boot from that virtual CD-ROM and wait for the installation to finish. Then, we will deploy CI runners on the management servers such that they can take deployment jobs from version control.
```

What is a preseed file?

Reply:
A preseed file on Debian derivatives is used to install the OS unattended.